src/errors.h

```diff
@@ -3508,6 +3508,14 @@ EXTERN char e_static_member_not_supported_in_interface[]
 	INIT(= N_("E1377: Static member is not supported in an interface"));
 EXTERN char e_method_str_of_class_str_has_different_access[]
 	INIT(= N_("E1378: Access level of method \"%s\" is different in class \"%s\""));
+EXTERN char e_static_cannot_be_used_in_interface[]
+	INIT(= N_("E1379: Static cannot be used in an interface"));
```

Comment:
E1377 is not used. Looks E1379 has taken it's place.

Reply:
I have used E1377 for some other message.